Author: Thierry Reding <thierry@gilfi.de>
Description: Enable snapping of windows by default in wobbly plugin.
 This patch originally by Thierry Reding, later modified/updated by
 Sean Finney. Not submitted to upstream.
Origin: vendor, debian (0.0.13+git20060928-1)
--- compiz.orig/metadata/wobbly.xml.in
+++ compiz/metadata/wobbly.xml.in
@@ -20,7 +20,7 @@
 	    <option name="snap_inverted" type="bool">
 		<_short>Snap Inverted</_short>
 		<_long>Inverted window snapping</_long>
-		<default>false</default>
+		<default>true</default>
 	    </option>
 	    <option name="shiver" type="bell">
 		<_short>Shiver</_short>
